tests/fsharp/tests.fs

```diff
@@ -757,70 +758,70 @@ module CoreTests =
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-5`` () = signedtest("test-sha1024-full-cl", "--keyfile:sha1024full.snk", SigningType.PublicSigned)
 
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-6`` () = signedtest("test-sha1-delay-cl", "--keyfile:sha1delay.snk --delaysign", SigningType.DelaySigned)
 
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-7`` () = signedtest("test-sha256-delay-cl", "--keyfile:sha256delay.snk --delaysign", SigningType.DelaySigned)
 
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-8`` () = signedtest("test-sha512-delay-cl", "--keyfile:sha512delay.snk --delaysign", SigningType.DelaySigned)
 
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-9`` () = signedtest("test-sha1024-delay-cl", "--keyfile:sha1024delay.snk --delaysign", SigningType.DelaySigned)
 
     // Test SHA1 key full signed  Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-10`` () = signedtest("test-sha1-full-attributes", "--define:SHA1", SigningType.PublicSigned)
 
     // Test SHA1 key delay signed  Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-11`` () = signedtest("test-sha1-delay-attributes", "--keyfile:sha1delay.snk --define:SHA1 --define:DELAY", SigningType.DelaySigned)
 
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-12`` () = signedtest("test-sha256-full-attributes", "--define:SHA256", SigningType.PublicSigned)
 
     // Test SHA 256 bit key delay signed  Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-13`` () = signedtest("test-sha256-delay-attributes", "--define:SHA256 --define:DELAY", SigningType.DelaySigned)
 
     // Test SHA 512 bit key fully signed  Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-14`` () = signedtest("test-sha512-full-attributes", "--define:SHA512", SigningType.PublicSigned)
 
     // Test SHA 512 bit key delay signed Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-15`` () = signedtest("test-sha512-delay-attributes", "--define:SHA512 --define:DELAY", SigningType.DelaySigned)
 
     // Test SHA 1024 bit key fully signed  Attributes
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
     let ``signedtest-16`` () = signedtest("test-sha1024-full-attributes", "--define:SHA1024", SigningType.PublicSigned)
 
     // Test fully signed with pdb generation
-    [<Test; Category("signedtest")>]
+    [<Fact; Trait("Category","signedtest")>]
```

Comment:
I don't think we account for this anywhere (cannot find any notion of this in the code), maybe you can just delete this trait for now.

Reply:
AFAIU even when this was added ([8 years ago](https://github.com/dotnet/fsharp/pull/1744)), this wasn't used, probably was some local ad-hoc thing.